website/client/src/components/groups/invitationListModal.vue

```diff
@@ -0,0 +1,271 @@
+<template>
+  <b-modal
+    id="invitation-list"
+    size="md"
+    :hide-header="true"
+    :hide-footer="true"
+  >
+    <div class="dialog-close">
+      <close-icon @click="close()" />
+    </div>
+    <h2 class="text-center textCondensed" v-once>
+      {{ $t('invitations') }}
+    </h2>
+    <div
+      v-for="member in members"
+      :key="member._id"
+      class="member-row"
+    >
+      <div class="class-icon">
+        <class-badge
+          v-if="member.stats"
+          :member-class="member.stats.class"
+          style="--badge-size: 40px"
+        />
+      </div>
+      <div class="usernames">
+        <user-label :user="member" class="user-label" /> <br>
+        <span class="username">
+          @{{ member.auth.local.username }}
+        </span>
+      </div>
+      <div :class="{
+        'status': true,
+        'accepted': member.accepted === true,
+        'declined': member.accepted === false,
+        'pending': member.accepted === null
+      }">
+        <div
+          v-if="member.accepted === true"
+          class="accepted float-right"
```

Comment:
A lot of elements could be `v-once` here, static translation strings and icons

Reply:
v-if and v-once wouldn't work together - should the label be in its own span with a v-once in it ?